woocommerce/woocommerce-android
# Add test coverage to OrderEditingViewModel

Summary
==========
Fixes issue #5119 by improving the unit test coverage for `OrderEditingViewModel` behavior.

How to Test
==========
Simply run the unit tests, try to mess with the `OrderEditingViewModel` states and verify if the respective tests fails or pass when expected.

Update release notes:
